rules/eslint.js

```diff
@@ -15,6 +15,7 @@ module.exports = {
     // lines being used.
     // Duplicate of @typescript-eslint/lines-between-class-members.
     'lines-between-class-members': 'off',
+    'max-lines': 'off',
```

Comment:
Let's add a comment explaining why, e.g. "This rule was initially enabled, however in practice it appears that developers can judge better if the file is too large or not, with many valid exceptions created over time.".